flask_mqtt/__init__.py

```diff
@@ -10,6 +12,59 @@
 
 __version__ = '0.0.10'
 
+TopicQos = namedtuple('TopicQos', ['topic', 'qos'])
+
+
+class TopicQosList(MutableSequence):
```

Comment:
Why do you use a MutableSequence here? As data doesn't get altered in my opinion a normal list would suffice here.

Reply:
By using MutableSequence, an abstract class in collections, rather than directly inheriting the list I thought it would be better to expand later. If list is enough, it seems to use list.